README.md

```diff
@@ -1,58 +1,39 @@
+| Python   | [![PyPI Latest Release](https://img.shields.io/pypi/v/onc.svg)](https://pypi.org/project/onc/)                                                                                         | [![License - Apache 2.0](https://img.shields.io/pypi/l/onc.svg)](https://github.com/OceanNetworksCanada/api-python-client/blob/main/LICENSE.txt)                         | [![GitHub Repo](https://img.shields.io/badge/api--python--client-gray?logo=github)](https://github.com/OceanNetworksCanada/api-python-client) | [![Sphinx Documentation](https://img.shields.io/badge/api--python--client-gray?logo=sphinx)](https://oceannetworkscanada.github.io/api-python-client/index.html) |
 
 ## Get started
-Follow these steps to get started.
-
-All requests require a unique Oceans 3.0 API token that authorizes you to access our data. 
-### How to obtain an ONC token?
 
- - Register for an Oceans 3.0 account at https://data.oceannetworks.ca/Registration
- - Log-in into your account at https://data.oceannetworks.ca  by clicking the Login link
- - Click the Profile link to access your account profile
- - Access the Web Services API tab and click "Generate Token"
+All requests require a unique Oceans 3.0 API token that authorizes you to access our data.
 
- - If you forget your token, you can always find it again in your Oceans 3.0 account profile.
-
-### How to get starteted with ONCs API?
-This is a tutorial to get you started:
-https://github.com/OceanNetworksCanada/api-tutorial 
-
-### Our existing client libraries (subject to change!)
-
- - [Matlab](https://github.com/OceanNetworksCanada/api-matlab-client)
- - [Python](https://github.com/OceanNetworksCanada/api-python-client)
- - [R](https://github.com/OceanNetworksCanada/api-r-client)
+### How to obtain an ONC token?
 
-### Some common code Examples
- 
- - [Matlab Example Codes](https://github.com/OceanNetworksCanada/onc-api-example-codes/tree/main/scripts/MATLAB)
- - [Python Example Codes](https://github.com/OceanNetworksCanada/onc-api-example-codes/tree/main/scripts/Python3)
- - [R Example Codes](https://github.com/OceanNetworksCanada/onc-api-example-codes/tree/main/scripts/R)
+- Register for an Oceans 3.0 account at https://data.oceannetworks.ca/Registration.
 
-## Credits
+- Log into your account at https://data.oceannetworks.ca by clicking the Log In link.
 
+- Click the Profile link (top right corner) to access your account profile.
 
-## License
+- Access the Web Services API tab and click Copy Token.
 
+- If you forget your token, you can always find it in your Oceans 3.0 account profile.
 
----
+### How to get started with Oceans 3.0 API?
 
+There are some [tutorials](https://oceannetworkscanada.github.io/api-python-client/Tutorial/index.html) to get you started.
 
+We suggest to start with how Oceans 3.0 API works (key concepts like discovery, data product delivery, real-time and archive files),
```

Comment:
We suggest to start with how Oceans 3.0 API works (key concepts like: data and instrument discovery, data product delivery, real-time data access and archived files),